miqcli/collections/__init__.py

```diff
@@ -24,8 +24,6 @@ class Collection(ClientAPI):
     Main option is to save the settings
     """
 
-    _settings = None
-
     def __init__(self, settings):
         """
         :param settings: MIQ settings
```

Comment:
It wont let me comment on it but could we remove the connect method from the Collection class? The collection class inherits the ClientAPI class which provides the connect method directly to the Collection class.

Reply:
agreed